tests/infer/mcmc/test_nuts.py

```diff
@@ -13,18 +14,19 @@
 from pyro.infer.mcmc.mcmc import MCMC
 from tests.common import assert_equal
 
-from .test_hmc import rmse, TEST_CASES, TEST_IDS
+from .test_hmc import rmse, T, TEST_CASES, TEST_IDS
 
 logging.basicConfig(format='%(levelname)s %(message)s')
 logger = logging.getLogger('pyro')
 logger.setLevel(logging.INFO)
 
 TEST_CASES[0] = TEST_CASES[0]._replace(mean_tol=0.04, std_tol=0.04)
 TEST_CASES[1] = TEST_CASES[1]._replace(mean_tol=0.04, std_tol=0.04)
```

Comment:
At this point, I feel that it might just be better to copy over the test cases. :) That will be more readable.

Reply:
Do you want to make that change now? I think that it is better to do it if there are more samplers come. This way makes the difference between HMC and NUTS easier to see: more precise for test 0, test 1, fewer samples for test 2 test 3.

Reply:
Makes sense. Let's punt on that now, but it might be worth thinking later how best to refactor these cases to support different variants like hmc, nuts or nuts with dual averaging.